acceptance_tests/docker-compose.yaml

```diff
@@ -8,8 +8,22 @@ services:
       MAPSERVER_CATCH_SEGV: 1
     links:
       - db
-    ports:
-      - 8380:8080
+    #ports:
+    #  - 8380:8080
+    volumes:
+      - ./config/mapserver/:/etc/mapserver
+    user: www-data
+
+  mapserver-map:
+    image: camptocamp/mapserver:${DOCKER_TAG}
+    environment:
+      MS_DEBUGLEVEL: 5
+      MAPSERVER_CATCH_SEGV: 1
+      MS_MAPFILE:
+    links:
+      - db
+    #ports:
+    #  - 8380:8080
     volumes:
       - ./config/mapserver/:/etc/mapserver
```

Comment:
change this location

Reply:
If I do that I can't test any more if the default MS_MAP_PATTERN is working...